tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php

```diff
@@ -97,11 +97,10 @@ private function test123(){
 EOF;
         $this->doTest($this->removeLinesFromString($template, $lineNumberRemoved), $template);
 
-        if (null === $config) {
-            $this->fixer->configure(array());
-        } else {
-            $this->fixer->configure(array('tokens' => $config));
-        }
+        null === $config
```

Comment:
Using a ternary expression to run code without using the returned value looks odd to me, can we use

```php
$this->fixer->configure(null === $config ? array() : array('tokens' => $config));
```
instead?